common/store.go

```diff
@@ -248,7 +251,10 @@ func createSchemaMysql(dbStore *DbStore, force bool) error {
 
 	for i := range entities {
 		entity := entities[i]
-		dbStore.Db.CreateTable(entity)
+		db := dbStore.Db.CreateTable(entity)
```

Comment:
This is inside of a loop. If one of the create-table operations fails, what happens to those that were already created. Is this atomic (they will be undone)? Or do they stay around?